pkg/generators/markers_test.go

```diff
@@ -167,315 +249,254 @@ func TestParseCommentTags(t *testing.T) {
 func TestCommentTags_Validate(t *testing.T) {
 
 	testCases := []struct {
-		name          string
-		commentParams map[string]any
-		t             types.Type
-		errorMessage  string
+		name         string
+		comments     []string
+		t            *types.Type
+		errorMessage string
 	}{
 		{
 			name: "invalid minimum type",
-			commentParams: map[string]any{
-				"minimum": 10.5,
+			comments: []string{
+				`+k8s:validation:minimum=10.5`,
```

Comment:
I think I missed the discussion on adding this prefix, is it going to cause any backwards incompatibilities with the rename?

Reply:
This isn't a rename, I changed the way the test works so that it expects the comments before parsing rather than after parsing